### ISSUE TITLE
Improve preprocessing normalisation

### ISSUE THREAD

**Issue by hyenal** [martinsbruveris/tensorflow-image-models#78]
Right now the preprocessing function does not account for uniform images. This can results in bugs which are hard to track.

I would suggest replacing this [function](https://github.com/martinsbruveris/tensorflow-image-models/blob/e9b22d1e0c20a50cf1459be534e37ef25da1ee92/tfimm/models/factory.py#L155), by the normalisation generally done in [tensorflow](https://github.com/keras-team/keras/blob/v2.11.0/keras/applications/efficientnet.py#L357) (`Normalisation` account for std =0).

**Comment by martinsbruveris** [comment]
I am bound to use the same preprocessing method as the one used for model training... And so I am copying whatever is present in `timm`.
